package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "Microverse project",
+  "main": "webpack.config.js",
+  "dependencies": {
+    "webpack": "^5.27.2",
+    "webpack-cli": "^4.5.0"
```

Comment:
[required] All webpack dependecies should be listed in devDependencies, [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from the pathwright